cmd/juno/juno.go

```diff
@@ -31,11 +31,13 @@ const (
 	dbPathF   = "db-path"
 	networkF  = "network"
 	pprofF    = "pprof"
+	colorF    = "color"
 
 	defaultConfig  = ""
 	defaultRPCPort = uint16(6060)
 	defaultDBPath  = ""
 	defaultPprof   = false
+	defaultColor   = false
```

Comment:
lets make it `true` by default

Reply:
So it means the output will be colorized by default? Unless set it to false to disable colorized output.

Reply:
Colored by default and `—color false` to disable. Would that work?

Reply:
Would `--color=false` be okay? Because I found this https://github.com/spf13/cobra/issues/613 that said cannot use `--bool [true|false]` nor the shorthand `-b[true|false]`.

Reply:
LGTM